airflow/providers/openlineage/plugins/adapter.py

```diff
@@ -329,19 +330,26 @@ def dag_started(
         self,
         dag_run: DagRun,
         msg: str,
-        nominal_start_time: str,
-        nominal_end_time: str,
         job_facets: dict[str, BaseFacet] | None = None,  # Custom job facets
+        run_facets: dict[str, BaseFacet] | None = None,  # Custom run facets
     ):
+        if not run_facets:
+            run_facets = {}
```

Comment:
Is this used anywhere now that we don't build this new run facet in listener? I think we can remove it for now if not used.

Reply:
Removed